packages/react/src/Grid/Grid.tsx

```diff
@@ -29,17 +29,28 @@ type GridPaddingTopAndBottomProps = {
 }
 
 export type GridProps = {
+  /** The HTML element to use. */
+  as?: 'header' | 'div' | 'footer'
```

Comment:
This breaks the forwardRef test because it's only for a div. Perhaps we needs something like this: https://medium.com/@yishayhaz/how-to-use-as-prop-in-react-typescript-2c310eb8ee1e

Reply:
I’ve updated to type the ref as `any` now, as we [decided a while ago](https://github.com/Amsterdam/design-system/pull/1140/files), but it looks like we can give the approach in that article a try.